modules/luci-compat/luasrc/tools/webadmin.lua

```diff
@@ -42,9 +42,9 @@ function date_format(secs)
 	end
 	
 	if days > 0 then
-		return string.format("%.0fd %02.0fh %02.0fmin %02.0fs", days, hour, mins, secs)
+		return string.format("%.0f d %02.0f h %02.0f min %02.0f s", days, hour, mins, secs)
 	else
-		return string.format("%02.0fh %02.0fmin %02.0fs", hour, mins, secs)
+		return string.format("%02.0f h %02.0f min %02.0f s", hour, mins, secs)
```

Comment:
What's the reason for extra spacing between the value and denominator?

Reply:
See my comment above.

Reply:
I'm not sure if the `2 h 2 m 2 s` is better from the readability standpoint than `2h 2m 2s`.

Reply:
It isn't. It may be right grammatically, but for status info the short version is much more understandable

Reply:
Well it would be `2 h 2 min 2 s`, which is the correct way to print it. The symbol for minute(s) is `min`, not `m`. It adds readability because the space separates the numbers from the units.